mappings/net/minecraft/client/gui/screen/Screen.mapping

```diff
@@ -57,7 +57,7 @@ CLASS net/minecraft/class_388 net/minecraft/client/gui/screen/Screen
 		ARG 1 button
 	METHOD method_14502 (Lnet/minecraft/class_1071;)Ljava/util/List;
 		ARG 1 stack
-	METHOD method_4259 mouseDragged (IIIJ)V
+	METHOD method_4259 mouseX (IIIJ)V
```

Comment:
this doesnt sound right

Reply:
It isn't, it's what happens when I mis-click on something while tired

Reply:
Feexed